### PR TITLE
Insecure exception with credential should raise warning

### PR DESCRIPTION
**Pull request by mabelvj**
If we are doing are getting the insecure credentials and we have username or password, an alert will raise.

### CODE REVIEW

File: shub/image/utils.py [hunk 1]
```diff
@@ -157,6 +157,9 @@ def get_credentials(username=None, password=None, insecure=False,
     without 'account' query parameter which breaks login.
     """
     if insecure:
+        if username or password:
+            raise click.BadParameter(
```
Comment (vshlapakov) [inline shub/image/utils.py]: Could you please replace it with [BadParameterException](https://github.com/scrapinghub/shub/blob/master/shub/exceptions.py#L57)? It's nearly the same as original `BadParameter`, but I would prefer to be consistent with the other code.

Comment (mabelvj) [reply shub/image/utils.py]: Sure!

Comment (vshlapakov) [reply shub/image/utils.py]: One thing, the `BadParameterException` lives in `shub.exceptions`, not in `click`

Comment (mabelvj) [reply shub/image/utils.py]: I'm checking right now and in the rest of the function it is used `BarParameter`. For consistency do you refer to other code in shub? Should the lines below 160 too? https://github.com/scrapinghub/shub/blob/6b71fb2ea378e8c659629219617739c9eda286c3/shub/image/utils.py#L168

Comment (vshlapakov) [reply shub/image/utils.py]: Oh, you're right indeed, I missed that. 
> For consistency do you refer to other code in shub?

Yeah, exactly. Could you please replace the other `click.BadParameter` in the function with `BadParameterException` from `shub.exceptions`?

**Comment by mabelvj** [comment]
ok, I will use the merge squash and combine them into a single one

**Comment by noviluni** [comment]
Hi @mabelvj, good job! if you need help with the testing don't hesitate to ask it :)

**Comment by mabelvj** [comment]
Thanks @Gallaecio @noviluni! I've just updated the PR with the tests.

**Comment by vshlapakov** [comment]
Awesome, thanks, merged! And sorry for the delay, I was on vacation :raised_hands:

**Pull request merged**